lms/static/js/courseware/bridge.js

```diff
@@ -0,0 +1,58 @@
+function sendMessageToIOS(message) {
+  window?.webkit?.messageHandlers?.IOSBridge?.postMessage(message);
+}
+
+function sendMessageToAndroid(message) {
+  window?.AndroidBridge?.postMessage(message);
+}
+
+function markProblemCompletedIOS(message) {
+  markProblemCompleted(message);
+}
+
+function markProblemCompletedAndroid(message) {
+  markProblemCompleted(message);
+}
+
+function markProblemCompleted(message) {
```

Comment:
Please add JSdoc to every function in this file

Like this - https://github.com/raccoongang/edx-platform/blob/63e940d65dbcd0191aa3d2865de6420d0f2b2c33/lms/static/js/student_account/emailoptin.js#L15

https://jsdoc.app/about-getting-started